sklearn/metrics/_classification.py

```diff
@@ -154,10 +154,16 @@ def _check_targets(y_true, y_pred):
         "y_pred": ["array-like", "sparse matrix"],
         "normalize": ["boolean"],
         "sample_weight": ["array-like", None],
+        "zero_division": [
+            Options(Real, {0, 1}),
```

Comment:
I think you have to include `np.nan` here.

Reply:
Done.